bin/trait_mapping/create_efo_table.py

```diff
@@ -79,7 +84,12 @@ def get_ols_details(ontology, term):
     # Cross-references
     term_curie = uri_to_curie(term)
     xrefs = {}
-    for x in get_cross_references(term_curie):
+    try:
+        oxo_xrefs = get_cross_references(term_curie)
+    except Exception as e:
```

Comment:
```suggestion
    except ValueError as e:
```
Unless you're expecting another exception to be raise ? 
You could also use the text set in the exception.